qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
+
+Scenario Outline: 1.0.0_01 All buttons of header lead to correct pages
+	When user clicks on <button>
+	Then <page> is opened
+
+	Examples:
+	|button            |page                                 |
+	|About             |https://career.quantori.com/about    |
+	|Open job positions|https://career.quantori.com/positions|
+	|Contact           |https://career.quantori.com/contact  |
+
+Scenario Outline: 1.0.0_02 All buttons of footer lead to correct pages
+	When user clicks on <button>
+	# VPN must be active
+	Then <page> is opened
+
+	Examples:
+	|button         |page                                                                             |
+	|planet         |https://www.quantori.com                                                         |
+	|postal envelope|https://mail.google.com/mail/u/0/?fs=1&tf=cm&source=mailto&to=career@quantori.com|
+	|hh             |https://hh.ru/employer/3343960                                                   |
+	|f              |https://www.facebook.com/quantoricom/                                            |
+	|camera         |https://www.instagram.com/quantori_community/                                    |
+	|vk             |https://vk.com/quantori                                                          |
+	|in             |https://www.linkedin.com/company/quantori                                        |
+
+Scenario Outline: 1.0.0_03 All the rest buttons of body lead to correct pages
```

Comment:
I would recommend to change the name for this scenario, make it more specific. May be, as we only have two buttons, just add the names of these buttons? Additionally, in the next scenario you also check the buttons in the body.

Reply:
Done) I combined the scenarios, so you can see the result in buttons_of_Career_Quantori_page.feature 1.0.0_03